opensearch/src/main/java/org/opensearch/sql/opensearch/data/value/OpenSearchExprValueFactory.java

```diff
@@ -164,7 +168,7 @@ public ExprValue construct(String jsonString, boolean supportArrays) {
           new OpenSearchJsonContent(OBJECT_MAPPER.readTree(jsonString)),
           TOP_PATH,
           Optional.of(STRUCT),
-          supportArrays);
+          fieldTypeTolerance || supportArrays);
```

Comment:
I'm thinking should we use `fieldTypeTolerance` directly in `parseArray` (not sure `parseGeopoint`)? Because I think `supportArrays` has its own meaning? Unless we change it and rename the argument?

Reply:
This also applies to geopoints and anything else that could be in an array. If `fieldTypeTolerance` is not also applied to `parseGeopoint`, then an array of geopoints would get reduced to only the first element.